Harvest.Web/Controllers/AppleController.cs

```diff
@@ -0,0 +1,67 @@
+﻿using Harvest.Core.Data;
+using Harvest.Core.Domain;
+using Harvest.Web.Models.Settings;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Options;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Harvest.Web.Controllers
+{
+    public class AppleController : Controller
+    {
+        private readonly AppDbContext _dbContext;
+        private readonly AuthSettings _authSettings;
+
+        public AppleController(AppDbContext dbContext, IOptions<AuthSettings> authSettings)
+        {
+            _dbContext = dbContext;
+            _authSettings = authSettings.Value;
+            if (string.IsNullOrEmpty(_authSettings.IPhoneKey) || _authSettings.IPhoneKey == "[External]")
```

Comment:
better to put inside the login method instead of ctor?  I'm worried it could break prod if we don't remember to add in the config instead of just breaking this one method

Reply:
Done, but controller got renamed, so isn't showing up here